ipaclient/install/client.py

```diff
@@ -2049,13 +2051,13 @@ def install_check(options):
 
     if options.conf_ntp:
         try:
-            timeconf.check_timedate_services()
-        except timeconf.NTPConflictingService as e:
+            ntpmethods.check_timedate_services()
+        except ntpmethods.NTPConflictingService as e:
             print("WARNING: conflicting time&date synchronization service '{}'"
                   " will be disabled".format(e.conflicting_service))
-            print("in favor of chronyd")
+            print("in favor of %s" % TIME_SERVICE)
```

Comment:
Please use .format()
```suggestion
            print("in favor of {}".format(TIME_SERVICE))
```